### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_RestoreDatabaseDialog_RestoreDatabaseGeneralTab_AI4D: Control type for General, Files & Options should be Tab

### ISSUE THREAD

**Issue by AbhiramDeshmukh** [microsoft/azuredatastudio#6695]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHub Tags: #A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #AI4D; #MAS4.1.2; #MAS4.2.1;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.10.0-insider (user setup)
Commit: 1ff9a2e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tool Name: Accessibility Insights tool
MAS References: MAS4.1.2, MAS4.2.1, MAS2.4.3

**Repro Steps:**

1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Expand the server connected in step 2.
5. Right click an on prem database and select Restore from Context menu.
6. Start Accessibility Insights for windows tool.
7. Check the control type provided for General, Files & Options tabs.

**Actual:**
Issue 1: Control type is given as "Tab Item" for General, Files & Options.
Issue 2: Keyboard user has to move using Keyboard TAB key between these tabs instead of arrow keys. Due to this, after General tab, keyboard focus moves to Files instead of going to Restore From combo box.

**Expected:**
For General, Files & Options, control type should be given as Tab. And for the complete group control type should be Tab list.

**User Impact:**
As control type is not listed as Tab, keyboard user has to navigate using TAB key between them.
Also after General, keyboard moves to in Files instead of moving to "Restore From" combo box which is resulting into extra tabs.

**Recommendation:**
Please refer to below URL which is repository for accessibility bug fixes.
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
MAS 4.1.2: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
MAS 4.2.1: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a12f3779-bdf1-40c2-bbcc-0e314ea94858}
MAS 2.4.3: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Attachment for reference:**
![6695_A11y_AzureDataStudio(Schema,Dacpac,Backup)_RestoreDatabaseDialog_RestoreDatabaseGeneralTab_AI4D_1](https://user-images.githubusercontent.com/31536050/62863637-01052780-bd27-11e9-928c-3dfa482afbee.png)
![6695_A11y_AzureDataStudio(Schema,Dacpac,Backup)_RestoreDatabaseDialog_RestoreDatabaseGeneralTab_AI4D_2](https://user-images.githubusercontent.com/31536050/62863638-01052780-bd27-11e9-8523-7cc454090b25.png)
[6695_A11y_AzureDataStudio(Schema,Dacpac,Backup)_RestoreDatabaseDialog_RestoreDatabaseGeneralTab_AI4D.pptx](https://github.com/microsoft/azuredatastudio/files/3492213/6695_A11y_AzureDataStudio.Schema.Dacpac.Backup._RestoreDatabaseDialog_RestoreDatabaseGeneralTab_AI4D.pptx)

**Comment by udeeshagautam** [comment]
Since this is for backup restore and we will not do Backup Restore GA in this wave we can close this? Following up...

**Comment by kenvanhyning** [comment]
Feature not moving to GA in current planning.  Closing as won't fix.
